apps/emqx_resource/src/emqx_resource_manager_sup.erl

```diff
@@ -15,6 +15,10 @@
 
 ensure_child(ResId, Group, ResourceType, Config, Opts) ->
     case supervisor:start_child(?MODULE, child_spec(ResId, Group, ResourceType, Config, Opts)) of
+        {error, {already_started, _}} ->
+            ok;
+        {error, already_present} ->
+            ok;
```

Comment:
I think those cases should not be silently ignored:  the resource id could exist with a different configuration than the one being passed here, hence it would lead to a silent "corruption" of the expected config.

If the plugin is starting the same resource id concurrently, it would be better to handle this error there than ignoring it for all resources, I believe.